spec/02-integration/05-proxy/05-dns_spec.lua

```diff
@@ -108,7 +108,7 @@ for _, strategy in helpers.each_strategy() do
 
         local service = bp.services:insert {
           name     = "tests-retries",
-          host     = "nowthisdoesnotexistatall",
+          host     = "nowthisdoesnotexistatall.test",
```

Comment:
The DNS server will reply "(2) server failure" for an A-type domain without dot, like "nowthisdoesnotexistatall".
Because we have removed CNAME from LAST_ORDER, the final attempt is now changed from CNAME to A.